src/serializer/functions.js

```diff
@@ -235,7 +236,9 @@ export class Functions {
     };
 
     let optimizedFunctionId = 0;
-    let getEffectsFromAdditionalFunctionAndNestedFunctions = (functionValue, argModelString) => {
+    let getEffectsFromAdditionalFunctionAndNestedFunctions = (functionValue, argModel) => {
```

Comment:
Confusing function name: It starts with `get`, but it doesn't actually return anything. What does it mutate?